lib/cn/csched_sp3.c

```diff
@@ -2546,6 +2561,8 @@ sp3_monitor(struct work_struct *work)
             sp3_schedule(sp);
         }
 
+        sp3_ucomp_check(sp);
```

Comment:
I don't this this should be here, ucomp_check gets called plenty via  sp3_update_samp() which is called a few lines above...

Reply:
The call site was moved from inside `sp3_update_samp()` to the main loop.  Not sure why.  Maybe there were cases that wasn't called when it was in `sp3_udpate_samp()`.